src/elements/content-sidebar/stories/tests/BoxAISidebar-visual.stories.tsx

```diff
@@ -25,11 +25,25 @@ export const BoxAIInSidebar: StoryObj<typeof BoxAISidebar> = {
     },
 };
 
-export const BoxAIWithExpandedView: StoryObj<typeof BoxAISidebar> = {
+export const BoxAIWithAgentSelector: StoryObj<typeof BoxAISidebar> = {
+    args: {
+        features: {
+            ...mockFeatures,
+            'boxai.agentSelector.enabled': true,
+        },
+    },
+    play: async ({ canvasElement }) => {
+        const canvas = within(canvasElement);
+        const agentSelector = await canvas.findByTestId('sidebar-agent-selector');
+        expect(agentSelector).toBeInTheDocument();
```

Comment:
can we add an action to click on the selector so it opens up? this would make more sense for the VRT